connectors/sources/confluence.py

```diff
@@ -210,6 +212,34 @@ async def paginated_api_call(self, url_name, **url_kwargs):
                 )
                 break
 
+    async def paginated_api_call_for_datacenter_syncrule(self, url_name, **url_kwargs):
```

Comment:
What does `_syncrule` suffix stand for here?

Reply:
I have kept it like this because method is specific to to datacenter sync rule only.